Source/CarthageKit/Dependency.swift

```diff
@@ -1,16 +1,21 @@
 import Foundation
 import Result
+import ReactiveSwift
+import ReactiveTask
 import Tentacle
 
 /// Uniquely identifies a project that can be used as a dependency.
 public enum Dependency {
 	/// A repository hosted on GitHub.com or GitHub Enterprise.
+	/// This is treated as case-insensitive.
 	case gitHub(Server, Repository)
 
 	/// An arbitrary Git repository.
+	/// This is treated as case-sensitive.
 	case git(GitURL)
```

Comment:
`GitURL`'s equality is based on `normalizedURLString` and there is no case-insensitive comparison or case conversion in that implementation.

- https://github.com/Carthage/Carthage/blob/a3573d94f97458473b2d8ae0282cd6d34e9908a2/Source/CarthageKit/GitURL.swift#L8-L46
- https://github.com/Carthage/Carthage/blob/a3573d94f97458473b2d8ae0282cd6d34e9908a2/Source/CarthageKit/GitURL.swift#L64-L66

Reply:
https://github.com/Carthage/Carthage/pull/2185/files#diff-d36e2d0823f4b902aeb9274469b117c0R123